plutus-core/plutus-core/src/PlutusCore/Evaluation/Machine/ExMemory.hs

```diff
@@ -50,9 +48,6 @@ abstractly specifiable. It's an implementation detail.
 -}
 
 type Plain f (uni :: GHC.Type -> GHC.Type) (fun :: GHC.Type) = f TyName Name uni fun ()
--- | Caches Memory usage for builtin costing
--- | NOT the amount of memory it cost to calculate this value.
-type WithMemory f (uni :: GHC.Type -> GHC.Type) (fun :: GHC.Type) = f TyName Name uni fun ExMemory
```

Comment:
I was tempted to remove `Plain` as well, but that's used in some of the tests.

Reply:
I wouldn't mind expanding that type synonym at the call site and removing it here. But I don't care much either way.